returnn/oggzip.py

```diff
@@ -28,6 +28,7 @@ def __init__(
         no_conversion=False,
         returnn_python_exe=None,
         returnn_root=None,
+        rqmt=None,
```

Comment:
I'd not put the rqmt in the constructor. It's OK to have them set later. (This is the default for many jobs)